### DISCUSSION TITLE
What is the biggest dinosaur that ever existed?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#713] [Galaxy Trivia] [answered]
1. Shantungosaurus
2. Maraapunisaurus
3. Titanosaur Patagotitan mayorum
4. Patagotitan
5. T-Rex

****Clue- lived over 100 million years ago

**Comment by ParanoidUser** [comment, marked as answer]
Great question! ❤️ It should be **Titanosaur Patagotitan Mayorum** discovered in 🇦🇷.

![image](https://github.com/ParanoidUser/yolo/assets/5120290/b3f086da-ce62-4d3e-ad8c-cac1b2bb38d2)
